contracts/TickerRegistry.sol

```diff
@@ -92,11 +108,33 @@ contract TickerRegistry is ITickerRegistry, Util, Pausable, RegistryUpdater, Rec
         require(msg.sender == securityTokenRegistry, "msg.sender should be SecurityTokenRegistry contract");
         require(registeredSymbols[symbol].status != true, "Symbol status should not equal to true");
         require(registeredSymbols[symbol].owner == _owner, "Owner of the symbol should matched with the requested issuer address");
-        require(registeredSymbols[symbol].timestamp.add(expiryLimit) >= now, "Ticker should not be expired");
+        require(registeredSymbols[symbol].expiryDate >= now, "Ticker should not be expired");
         registeredSymbols[symbol].tokenName = _tokenName;
         registeredSymbols[symbol].status = true;
         return true;
     }
+   
+    /**
+     * @notice Register the ticker without paying the fee 
+       Once the token symbol is registered to its owner then no other issuer can claim
+       Its ownership. If the symbol expires and its issuer hasn't used it, then someone else can take it.
+     * @param _symbol token symbol
+     * @param _tokenName Name of the token
+     * @param _swarmHash Off-chain details of the issuer and token
+     * @param _registrationDate Date on which ticker get registered
+     * @param _expiryDate Expiry date of the ticker
+     */
+    function addCustomTicker(string _symbol, string _tokenName, bytes32 _swarmHash, uint256 _registrationDate, uint256 _expiryDate) public onlyOwner {
```

Comment:
`addCustomTicker` would be used to register a ticker on behalf of someone else, so one parameter should be `_owner` and not use msg.sender

Reply:
Do we think to use that function to migrate the tickers? I have thought a more generalize way where `polymathAccount` add the ticker then transfer the ownership of the ticker.

Reply:
I don't see the point in registering them to Polymath and then transferring ownership, vs straight register them to someone else. This would have come in handy for the Polycon regs we did, so we didn't need to send POLY alongside. And also, very useful to migrate data later on.